zerver/management/commands/send_realm_reactivation_email.py

```diff
@@ -0,0 +1,24 @@
+
+from argparse import ArgumentParser
+
+from zerver.lib.management import ZulipBaseCommand
+from zerver.lib.send_email import send_email, FromAddress
+from zerver.lib.actions import do_send_realm_reactivation_email
+
+from typing import Any
+
+class Command(ZulipBaseCommand):
+    help = """Sends realm reactivation email to admins"""
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        self.add_realm_args(parser, True)
+
+    def handle(self, *args: Any, **options: str) -> None:
+        realm = self.get_realm(options)
+        assert realm is not None
+        if not realm.deactivated:
+            print("The realm", realm.name, "is already active.")
+            exit(0)
```

Comment:
This should `raise CommandError()` instead; that does the same thing in terms of printing but is cleaner to test (no need to catch SystemExit).